app/javascript/components/request-workflow-status/data.js

```diff
@@ -43,13 +43,43 @@ const convertDate = (date) => {
   return formattedDate.toString();
 };
 
+// Converts the duration time in ms and returns a string in format: w days x hours y minutes z seconds
+// duration: time in ms
+const convertDuration = (duration) => {
+  const durationString = moment.duration(duration, 'milliseconds').toISOString().split('PT')[1];
```

Comment:
I'm very surprised this doesn't "throw away" the days, months, years component of the time